tests/dummy/app/router.js

```diff
@@ -17,6 +17,7 @@ Router.map(function () {
     this.route('tab')
     // Tests
     this.route('hook')
+    this.route('block-format')
```

Comment:
Should probably be in one of the section above and not under test. I tried to make this match with the overview page.